cross/android/arm64/tryrun.cmake

```diff
@@ -0,0 +1,128 @@
+# Need to verify this
+SET( HAVE_COMPATIBLE_EXP_EXITCODE 
```

Comment:
@cydhaselton Could you paste the contents of `bin/obj/Linux.arm64.Debug/CMakeCache.txt` in pastbin? That would allow me to verify the contents of this files with what you're seeing in the Termux environment.